samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist-kfpv2.ipynb

```diff
@@ -0,0 +1,777 @@
+    "    }\n",
+    "\n",
+    "    output = NamedTuple('Outputs', [('inferenceservice_yaml', Dict[str, str])])\n",
+    "    return output(inference_service)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This container component is KServe launcher, its API is same as the following yaml file in KFPv1.\n",
+    "# https://raw.githubusercontent.com/kubeflow/pipelines/master/components/kserve/component.yaml\n",
+    "from kfp.dsl import Input, Output, Artifact, container_component, ContainerSpec\n",
+    "\n",
+    "@dsl.container_component\n",
+    "def serving_launcher(action: str, inferenceservice_yaml: Dict[str, str]):\n",
+    "    return ContainerSpec(\n",
+    "        image='quay.io/aipipeline/kserve-component:v0.7.0',\n",
+    "        command=['python', 'kservedeployer.py' ],\n",
+    "        args=[\n",
+    "          '--action',                 action,\n",
+    "          '--inferenceservice-yaml',  inferenceservice_yaml,\n",
+    "        ])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@dsl.component\n",
+    "def convert_inference_service_to_artifact(inferenceservice_yaml: Dict[str, str], inferenceservice_artifact: Output[Artifact]):\n",
+    "    import json\n",
+    "    with open(inferenceservice_artifact.path, 'w') as f:\n",
+    "        f.write(json.dumps(inferenceservice_yaml))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@dsl.pipeline\n",
+    "def serving_pipeline(model_name: str, model_namespace: str, model_volume_name: str)    -> Artifact:\n",
+    "    create_serving_task_op = create_serving_task(model_name=model_name, model_namespace=model_namespace, model_volume_name=model_volume_name)\n",
+    "    convert_inference_service_to_artifact_op = convert_inference_service_to_artifact(\n",
+    "        inferenceservice_yaml=create_serving_task_op.outputs['inferenceservice_yaml'])\n",
+    "    serving_launcher_op = serving_launcher(\n",
+    "        action='apply', \n",
+    "        inferenceservice_yaml=create_serving_task_op.outputs['inferenceservice_yaml']\n",
+    "    ).after(convert_inference_service_to_artifact_op)\n",
+    "    \n",
+    "    return convert_inference_service_to_artifact_op.outputs['inferenceservice_artifact']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run the Kubeflow Pipeline\n",
+    "\n",
+    "You should create the Kubeflow Pipeline from the above tasks."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "name=\"mnist-e2e\"\n",
+    "namespace=\"kubeflow-user-project\"\n",
+    "training_steps=\"200\"\n",
+    "model_volume_name=\"workflow1-model-volume\"\n",
+    "\n",
+    "\n",
+    "import json\n",
```

Comment:
nit: `json` not used after this cell

Reply:
Good point! Cleaned up unnecessary `import json`